web_action_conditionable/README.rst

```diff
@@ -0,0 +1,94 @@
+========================
+web_action_conditionable
+========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/15.0/web_action_conditionable
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-15-0/web-15-0-web_action_conditionable
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Add support for conditions on create and delete actions on One2Many fields.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Odoo by default support:
+
+::
+
+   <tree delete="false" create="false">
+
+with this module you can:
+
+::
+
+   <tree delete="state=='draft'" create="state!='sent'">
```

Comment:
@jasperjumelet Hello, this module doesn't work for me, have a look at the image below:
- i have add `delete="state=='draft'" create="state!='sale'"` to the tag `tree` of  Sales Order Lines: `<tree string="Sales Order Lines" editable="bottom" delete="state=='draft'" create="state!='sale'">`
- My test Order `S00006` is in state of `Sales Order` (sale). But the button `Add a product` still be there.
![Screenshot from 2023-04-27 14-47-43](https://user-images.githubusercontent.com/8724793/234796175-f75ac1ab-27f6-489d-b4d4-8e48067bfea0.png)

Reply:
btw, please make `runboat/build` work.

Reply:
Hello @jasperjumelet i have created this [pull request](https://github.com/jasperjumelet/web/pull/1) to make this module worked. Please review. 
Thanks,

Reply:
Hello @StephaneMangin the new PR https://github.com/OCA/web/pull/2500 is the result of https://github.com/OCA/web/pull/2435 + https://github.com/jasperjumelet/web/pull/1 stashed in one single commit